<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangeNotice</class>
 <widget class="QMainWindow" name="ChangeNotice">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>480</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(136, 202, 94);</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>651</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 27pt &quot;MS Sans Serif&quot;;
background-color: rgb(255, 255, 255);
text-align: center;
border-radius:5px;

</string>
    </property>
    <property name="text">
     <string>How much money are you inserting?</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="amountExpected">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>110</y>
      <width>501</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 22pt &quot;MS Sans Serif&quot;;
background-color: rgb(255, 255, 255);
border-radius:5px;</string>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QFrame" name="numpadFrame">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>200</y>
      <width>221</width>
      <height>221</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 0, 127);
background-color: rgb(255, 255, 255);
</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QPushButton" name="one_key">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>120</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="two_key">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>120</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="three_key">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>120</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="four_key">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="five_key">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="six_key">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>70</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="seven_key">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="eight_key">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);
</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="nine_key">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>20</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="zero_key">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);</string>
     </property>
     <property name="text">
      <string>0</string>
     </property>
    </widget>
    <widget class="QPushButton" name="backspace_key">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>170</y>
       <width>41</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(217, 217, 217);
font: 15pt &quot;MS Sans Serif&quot;;
alternate-background-color: rgb(132, 132, 132);</string>
     </property>
     <property name="text">
      <string>⌫</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="proceed_Button">
    <property name="geometry">
     <rect>
      <x>660</x>
      <y>400</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 15pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Proceed</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame_5">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>440</y>
      <width>811</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);</string>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>310</x>
       <y>10</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">color: rgb(136, 202, 94);
font: 75 6pt &quot;MS Sans Serif&quot;;</string>
     </property>
     <property name="text">
      <string>© 2024 LMMN, Inc. All rights reserved</string>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="back_button">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>131</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">
font: 75 15pt &quot;MS Sans Serif&quot;;
border-radius: 10px; 
background-color: rgb(255, 255, 255);
color: rgb(136, 202, 94);</string>
    </property>
    <property name="text">
     <string>Back</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
